bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -736,3 +710,9 @@ private constructor(
         }
     }
 }
+
+private fun getConfiguredJson(): Json = Json {
```

Comment:
This code appears in multiple places and could be deduplicated a bit I guess.

Reply:
See the “Still open” section in the PR description 😊